src/auth-service/controllers/create-token.js

```diff
@@ -1,13 +1,43 @@
 const httpStatus = require("http-status");
 const controlAccessUtil = require("@utils/control-access");
 const { extractErrorsFromRequest, HttpError } = require("@utils/errors");
+const { logObject, logText } = require("@utils/log");
 const isEmpty = require("is-empty");
 const constants = require("@config/constants");
 const log4js = require("log4js");
 const logger = log4js.getLogger(
   `${constants.ENVIRONMENT} -- create-token-controller`
 );
 
+function handleResponse({
```

Comment:
@BenjaminSsempala , we shall start to use this to handle all our responses in the near future. 
Just to keep our controllers neat and tidy. :)